bigbluebutton-html5/imports/ui/components/breakout-room/create-breakout-room/room-managment-state/component.tsx

```diff
@@ -245,6 +247,39 @@ const RoomManagmentState: React.FC<RoomManagmentStateProps> = ({
     }
   }, [users]);
 
+  useEffect(() => {
+    if (groups.length && init && lastBreakoutData && !(lastBreakoutData.breakoutRoom_createdLatest.length > 0)) {
+      const sortedGroups = Array.from(groups).sort((a, b) => {
+        const aId = Number.parseInt(a.groupId, 10);
+        const bId = Number.parseInt(b.groupId, 10);
+
+        return aId - bId;
+      });
```

Comment:
It shouldn't order by group ID since the ID can be a hash and may not follow a sequence. I added a new field, `groupIndex`, for sorting. I’ve sent the commit to your PR!

![image](https://github.com/user-attachments/assets/0bb23139-f711-45d0-8790-9005692dc2eb)